rpc/types_transaction_receipt.go

```diff
@@ -271,108 +269,6 @@ func (tr L1HandlerTransactionReceipt) GetExecutionStatus() TxnExecutionStatus {
 	return tr.ExecutionStatus
 }
 
-type PendingL1HandlerTransactionReceipt struct {
```

Comment:
The pending transaction receipt is still valid as you can provide a tx hash which is pending.

Reply:
Sure, but pending receipts are no longer explicit types in rc-2 though.

[[starknet_getTransactionReceipt](https://github.com/starkware-libs/starknet-specs/blob/v0.7.0-rc2/api/starknet_api_openrpc.json#L367) explicitly returns [TXN_RECEIPT_WITH_BLOCK_INFO](https://github.com/starkware-libs/starknet-specs/blob/v0.7.0-rc2/api/starknet_api_openrpc.json#L3074) which returns [TXN_RECEIPT](https://github.com/starkware-libs/starknet-specs/blob/v0.7.0-rc2/api/starknet_api_openrpc.json#L3049), and not a PENDING_TXN_RECEIPT ](https://github.com/starkware-libs/starknet-specs/blob/v0.7.0-rc1/api/starknet_api_openrpc.json#L391) like it used to

Reply:
I see